modules/mssql/variables.tf

```diff
@@ -59,7 +71,7 @@ variable "edition" {
 variable "zone" {
   type        = string
   description = "The zone for the master instance."
-  default     = "us-central1-a"
```

Comment:
This will be a breaking change as well. If so, can you add details to the upgrade doc?